orbit/pkg/update/update.go

```diff
@@ -528,18 +528,17 @@ func (u *Updater) download(target, repoPath, localPath string, customCheckExec f
 		return err
 	}
 
+	downloadFilePath := filepath.Join(staging, filepath.Base(localPath))
+
 	tmp, err := secure.OpenFile(
-		filepath.Join(staging, filepath.Base(localPath)),
+		downloadFilePath,
 		os.O_CREATE|os.O_WRONLY,
 		constant.DefaultExecutableMode,
 	)
 	if err != nil {
 		return fmt.Errorf("open temp file for download: %w", err)
 	}
-	defer func() {
-		tmp.Close()
-		os.Remove(tmp.Name())
-	}()
```

Comment:
Why are we removing this?

Reply:
I forget exactly why since it was all done super hacky, I think I removed the `os.Remove` to stop it from deleting the downloaded file and keep it in the cache directory. This could definitely use a little TLC before it gets merged